dspace-server-webapp/src/test/java/org/dspace/app/rest/ClarinHandleImportControllerIT.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.dspace.app.rest.model.HandleRest;
+import org.dspace.app.rest.test.AbstractControllerIntegrationTest;
+import org.dspace.handle.Handle;
+import org.dspace.handle.service.HandleClarinService;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.web.servlet.MvcResult;
+
+/**
+ * Integration test to test the /api/clarin/import/handle endpoint
+ *
+ * @author Michaela Paurikova (michaela.paurikova at dataquest.sk)
+ */
+public class ClarinHandleImportControllerIT extends AbstractControllerIntegrationTest {
+    @Autowired
+    private HandleClarinService handleService;
+
+    @Test
+    public void createHandleTest() throws Exception {
```

Comment:
Please describe better what are you testing. Is it that handles without DSpace obejct?